rules/library.bzl

```diff
@@ -694,6 +695,7 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
         resource_bundles = kwargs.pop("resource_bundles", {}),
         module_name = module_name,
         platforms = platforms,
+        bundle_id = kwargs.pop("bundle_id", None),
```

Comment:
In the change on line 178 `bundle_id` was added to the function arguments, I expected this would no longer be present in `kwargs`. e.g. this line should just access it via the local var `bundle_id`?

See other comment about setting this to the frameworks bundle ID and better constructing the id internally

Reply:
I think the fact that the definition happens way up is creating the confusion here, `bundle_id` was added to the macro that is set to `library_tools["resource_bundle_generator"]` here. 

Meaning that the param `bundle_id` doesn't exist at this point yet and this `pop` is doing exactly that and setting the `bundle_id` param. Maybe open the complete file to inspect and see what I'm talking about?